client/document/browsehappy.jsx

```diff
@@ -24,17 +34,22 @@ function Browsehappy( { entrypoint, dashboardUrl } ) {
 									<EmptyContent
 										illustration="/calypso/images/drake/drake-browser.svg"
 										title="Unsupported Browser"
-										line={ [
-											'Unfortunately this page cannot be used by your browser. You can either ',
-											<a key="dashboard-link" href={ dashboardUrl }>
-												use the classic WordPress dashboard
-											</a>,
-											', or ',
-											<a key="browsehappy-link" href="https://browsehappy.com">
-												upgrade your browser
-											</a>,
-											'.',
-										] }
+										line={
+											<>
+												<p>
+													Unfortunately this page cannot be used by your browser. You can either{ ' ' }
```

Comment:
"cannot be used" seems a bit hash, and potentially misleading. How about "may not work correctly with" instead?